lib/src/models/region_info.dart

```diff
@@ -3,6 +3,9 @@ import 'package:meta/meta.dart' show immutable;
 /// Phone number region information
 @immutable
 class RegionInfo {
+  /// Localized name for the country.
```

Comment:
Sorry the comment isn't as descriptive as the others. Not sure where you got those...

Reply:
No worries, that's fine ;)